src/Microsoft.DotNet.CodeFormatting/Rules/RemoveRegionsRule.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright(c) Microsoft.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.DotNet.CodeFormatting.Rules
+{
+    [SyntaxRule(RemoveRegionsRule.Name, RemoveRegionsRule.Description, SyntaxRuleOrder.RemoveRegionsRule)]
```

Comment:
I believe this rule should be marked as `DefaultRule=false`.  I agree some people will find value in this rule but it's not a part of our explicit coding guidelines and it would be a great surprise if all regions were suddenly deleted from a lot of projects.